lesson-01/task-05.js

```diff
@@ -0,0 +1,40 @@
+let chinaPrice = 100;
+let chilePrice = 250;
+let australiaPrice = 170;
+let indiaPrice = 80;
+let jamaicaPrice = 120;
+
+let country = prompt("What country are you living in?");
+
+let capitalizedCountry =
+  country[0].toUpperCase() + country.slice(1).toLowerCase();
+
+switch (capitalizedCountry) {
+  case "China":
```

Comment:
У випадку попередньої зміни, назви країн просто потрібно написати заголовними літерами) 
```suggestion
  case "CHINA":
```